install_builder/deadline-cloud-for-unreal-engine.xml

```diff
@@ -18,24 +18,24 @@
             </distributionFileList>
         </folder>
         <folder>
-            <description>Dependency Files</description>
-            <destination>${unreal_plugin_dir}/tmp/unreal_deps</destination>
-            <name>unreal_deps</name>
+            <description>Logger Files</description>
+            <destination>${unreal_plugin_dir}/Content/Python/libraries/deadline/unreal_logger</destination>
+            <name>unreal_logger</name>
             <platforms>all</platforms>
             <distributionFileList>
                 <distributionDirectory allowWildcards="1">
-                    <origin>components/deadline-cloud-for-unreal-engine/dependency_bundle</origin>
+                    <origin>components/deadline-cloud-for-unreal-engine/src/deadline/unreal_logger/*</origin>∂
                 </distributionDirectory>
             </distributionFileList>
         </folder>
         <folder>
-            <description>Prebuild Outputs</description>
-            <destination>${unreal_plugin_dir}</destination>
-            <name>unreal_prebuilds</name>
+            <description>Dependency Files</description>
+            <destination>${unreal_plugin_dir}/tmp/unreal_deps</destination>
+            <name>unreal_deps</name>
             <platforms>all</platforms>
             <distributionFileList>
                 <distributionDirectory allowWildcards="1">
-                    <origin>components/deadline-cloud-for-unreal-engine/UnrealDeadlineCloudService/*</origin>
```

Comment:
Do we not need the files from here anymore?

Reply:
No we're currently not using this so I wanted to remove it until/unless we are.  The prebuilds are keyed to the Unreal version so for each new patch version of Unreal that's released (Which they may do up to weekly after releasing a new minor version release like they just did with 5.5) we'll often be out of date before our next submitter installer is even released currently.  We're investigating other options for this.